test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
I would suggest that the users ids should be generated in the server, datastore can do that for you, but be advised ```Each ID can be up to 16 decimal digits long.``` which would require you to use long for the id's